test/processor_test.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "test_helper"
```

Comment:
Let's add a simple test case to the existing tests instead of introducing new files.